server/handler.go

```diff
@@ -920,3 +906,17 @@ func (h *Handler) SetStoreLimitTTL(data string, value float64, ttl time.Duration
 		data: value,
 	}, ttl)
 }
+
+func checkStoreState(rc *cluster.RaftCluster, storeID uint64) error {
+	store := rc.GetStore(storeID)
+	if store == nil {
+		return errs.ErrStoreNotFound.FastGenByArgs(storeID)
+	}
+	if store.IsTombstone() {
```

Comment:
Maybe we can add one more argument in `ErrStoreUnhealthy` to distinguish these different situations.